ArcGISRuntimeSDKQt_CppSamples/Scenes/OrbitCameraAroundObject/OrbitCameraAroundObject.qml

```diff
@@ -0,0 +1,210 @@
+// [WriteFile Name=OrbitCameraAroundObject, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import Esri.Samples 1.0
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.12
+
+Item {
+
+    SceneView {
+        id: view
+        anchors.fill: parent
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    OrbitCameraAroundObjectSample {
+        id: model
+        sceneView: view
+        planePitch: planePitchSlider.value
+    }
+
+    //Color for the panels that the UI controls sit on.
+    property color uiBackgroundCol : Qt.rgba(0.2, 0.2, 0.2, 0.65)
+
+    //Camera heading slider, sits at the bottom of the screen
+    Rectangle {
+        anchors {
+            bottom: parent.bottom
+            left: parent.left
+            right: parent.right
+            margins: 30
+        }
+
+        height: cameraHeadingSlider.height + cameraHeadingLabel.height
+        color: uiBackgroundCol
```

Comment:
This sample looks different in terms of style to the other samples so far. Is it possible to bring the styling more in line?